Tests/Unit/RealMiniAppViewTests.swift

```diff
@@ -0,0 +1,20 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+class RealMiniAppViewTests: QuickSpec {
+
+    override func spec() {
+        describe("Mini App view") {
+            context("when initialized with valid parameters") {
```

Comment:
What about invalid?
Unit tests should cover the "unhappy" paths as well as the "happy" paths